package.json

```diff
@@ -58,7 +58,8 @@
     "ts:webdriverio-reporter": "cd tests/typings/webdriverio-reporter && tsc --incremental",
     "watch": "node ./scripts/watch",
     "watch:pkg": "lerna exec 'npm run compile -- --watch' --scope",
-    "version": "npm run changelog && git add CHANGELOG.md"
+    "version": "npm run changelog && git add CHANGELOG.md",
+    "postinstall": "git config --local include.path '../.gitconfig'"
```

Comment:
```suggestion
    "postinstall": "git config --local include.path './.gitconfig'"
```

Reply:
It should be .. because the path is relative to . git folder in the repo. We should go up to be in the project root.

Reply:
Yes, the . gitconfig is in the repo root same as .git but the base path for any config path is relative to projectRoot/.git.
So we should go up to the project root from .git folder

Reply:
Yes, the . gitconfig is in the repo root same as .git but the base path for any config path is relative to projectRoot/.git.
So we should go up to the project root from .git folder